### DISCUSSION TITLE
Chart File Format Similar To StepMania

### DISCUSSION THREAD

**Discussion by Johferson** [FunkinCrew/Funkin#1951] [Ideas]
because the JSON file formats quite suck and has a lot of strings going on ngl.

**Comment by isakube** [comment]
> because the JSON file formats quite suck and has a lot of strings going on ngl.

no lol

**Reply by Johferson** [reply]
debatable, but you got to agree that the JSON format used has a lot of strings and probably large, and StepMania have their Stepfiles minimalized into variables which translates into a Note or a Mine

**Reply by isakube** [reply]
easier to parse lmao
